noggin/form/base.py

```diff
@@ -65,3 +66,19 @@ class SubmitButtonField(SubmitField):
 
 def strip(value):
     return value.strip() if value else value
+
+
+class CSVListField(Field):
+    widget = TextInput()
+
+    def _value(self):
+        if self.data:
+            return ','.join(self.data)
+        else:
+            return ''
+
+    def process_formdata(self, values):
+        if values:
```

Comment:
I think you should test on `value.strip()`, otherwise if you get a string with only spaces you'll end up with `[""]` instead of an empty list. (feel free to add a unit test for that case ;-) )